src/RadioButtonGroup/RadioButtonGroup.stories.jsx

```diff
@@ -38,6 +42,9 @@ const onKeyDown = (event) => {
 };
 
 storiesOf('RadioButtonGroup', module)
+  .addDecorator((story, context) => withInfo({}, README)(story)(context))
+  .addDecorator(centered)
+  .addDecorator(checkA11y)
```

Comment:
You can add these globally but then it might show accessibility errors that we'd like to resolve in chunks (like this).

My rollout plan is to do these in pieces and then when we institute these globally to, as part of that PR, remove these storybook decorators.